src/main/java/com/google/jepsenonspanner/verifier/knossos/Node.java

```diff
@@ -0,0 +1,270 @@
+package com.google.jepsenonspanner.verifier.knossos;
```

Comment:
Could you please search for "conf" in this file and rename corresponding ones to"node"?